Description: Updated to python3
 Python 2 is being end of life. this file is python3
 compatible.
Author: Philippe Thierry <phil@reseau-libre.net>
--- a/utils/kconfig-diff
+++ b/utils/kconfig-diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 #
 # diffconfig - a tool to compare .config files.
 #
